packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1169,8 +1169,12 @@ export class TransactionController extends BaseController<
           return [meta, false];
         }
 
+        const { gasUsed, baseFeePerGas } = txReceipt;
```

Comment:
If I understood correctly, we should check if `txReceipt.blockNumber` exist, then another query to `getBlockByHash` to read that `baseFeePerGas` for that block.

Reply:
You are right, I thought I pushed this when I was cleaning up the commits. Anyway, it's pushed now, thanks!